cty/function/stdlib/encoding.go

```diff
@@ -0,0 +1,140 @@
+package stdlib
```

Comment:
About this file; do you think I should remove any function in it ?

If not, what about `Base64Sha256Func`, `Base64Sha512Func` ? Should I put them in here as well ?

Reply:
There is some precedent for encoding functions in this package (e.g. `JSONEncode`, `CSVDecode`, etc), but the ones there so far were motivated primarily by them being intuitive encodings of the `cty` data types, and in most cases also just wrappers around functionality that's already in this codebase anyway. I'd also previously drawn the line of not bringing YAML encoding in here and putting that in [a separate codebase](https://github.com/zclconf/go-cty-yaml).

For that reason, my instinct is to put the hashing functions alongside the other "crypto" stuff, and also to put the other encoding-ish things in a separate place because they aren't really `cty`-type-specific sorts of things, but are rather motivated by use-cases inside Terraform and now Packer.

Reply:
Got it; I understand and agree; I think I'll also create an encoding pkg 🤔